schemas/appliance_v8.json

```diff
@@ -0,0 +1,518 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "JSON schema validating a GNS3 appliance",
+
+  "definitions": {
+    "dynamips_slot": {
+        "enum": ["C2600-MB-2FE", "C2600-MB-1E", "PA-A1", "PA-8E", "C1700-MB-1FE", "PA-8T", "PA-2FE-TX", "PA-FE-TX", "PA-GE", "C2600-MB-2E", "C7200-IO-FE", "NM-4T", "C2600-MB-1FE", "C7200-IO-2FE", "PA-POS-OC3", "PA-4T+", "C1700-MB-WIC1", "NM-16ESW", "C7200-IO-GE-E", "NM-4E", "GT96100-FE", "NM-1FE-TX", "Leopard-2FE", "NM-1E", "PA-4E", ""]
+    },
+
+    "dynamips_wic": {
+        "enum": ["WIC-1ENET", "WIC-1T", "WIC-2T", ""]
+    }
+  },
+
+  "properties": {
+    "appliance_id": {
+      "title": "Appliance ID",
+      "type": "string",
+      "minLength": 36,
+      "maxLength": 36,
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
+    },
+    "name": {
+      "type": "string",
+      "title": "Appliance name"
+    },
+    "category": {
+      "enum": [
+          "router",
+          "multilayer_switch",
+          "firewall",
+          "guest"
+      ],
+      "title": "Category of the appliance"
+    },
+    "description": {
+      "type": "string",
+      "title": "Description of the appliance. Could be a marketing description"
+    },
+    "vendor_name": {
+      "type": "string",
+      "title": "Name of the vendor"
+    },
+    "vendor_url": {
+      "type": "string",
+      "format": "uri",
+      "title": "Website of the vendor"
+    },
+    "documentation_url": {
+      "type": "string",
+      "format": "uri",
+      "title": "An optional documentation for using the appliance on vendor website"
+    },
+    "product_name": {
+      "type": "string",
+      "title": "Product name"
+    },
+    "product_url": {
+      "type": "string",
+      "format": "uri",
+      "title": "An optional product url on vendor website"
+    },
+    "registry_version": {
+      "enum": [1, 2, 3, 4, 5, 6],
```

Comment:
Same issue as with appliance_v7.json: The allowed registry_version doesn't include the version 7 (appliance_v7.json + appliance_v8.json) and 8 (appliance_v8.json).